### PR TITLE
Disable javascript and other links from webview

### LINKED ISSUE #195: Sandbox Webview
Ensure the webview for the measurements page doesn't leak personal information, or open any other link inside the app.

### PR DESCRIPTION
**Pull request by sdsantos**
Closes #195

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt [hunk 1]
```diff
@@ -45,7 +49,22 @@ fun MeasurementScreen(
     LaunchedEffect(url) { Logger.i("URL: $url") }
 
     val webViewState = rememberWebViewState(url)
-    val webViewNavigator = rememberWebViewNavigator()
+    webViewState.webSettings.isJavaScriptEnabled = false
```
Comment (aanorbel) [inline composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt:52]: By disabling javascript, we lose the ability to view raw measurement data.
Screenshot_20241022_100007

**Review by aanorbel — CHANGES_REQUESTED** [review]
Re-enable javascript.

File: composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt (hunk 1 above)
Comment (sdsantos) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt:52]: @aanorbel I noticed, but those pages will eventually have a mobile friendly view, that won't have any link or interaction, according to @hellais. That's why I went with this change.

Comment (aanorbel) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt:52]: https://github.com/ooni/explorer/issues/951 this is the issue on explorer

Comment (sdsantos) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/ui/measurement/MeasurementScreen.kt:52]: Replied there with a comment 👍

**Pull request merged**